src/index.html

```diff
@@ -30,7 +30,7 @@
 
     <!-- for Facebook -->
     <meta property="og:title" content="CARE's World of Impact" />
-    <meta property="og:image" content="" />
+    <meta property="og:image" content="http://care-usa.herokuapp.com/care-map.jpg" />
```

Comment:
OK, let's not try:

"Documentation have nothing about it, but relative URLs will not work (as for me it's a time to file a bug), only full URL (including scheme, which is by design) works."